src/reports/components/report-sections/report-footer.tsx

```diff
@@ -2,28 +2,16 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { NewTabLink } from 'common/components/new-tab-link';
 import { NamedFC } from 'common/react/named-fc';
-import { toolName } from 'content/strings/application';
 import { SectionProps } from './report-section-factory';
 
-export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'>;
+export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'> & Pick<SectionProps, 'footerText'>;
 
-export const ReportFooter = NamedFC<ReportFooterProps>('ReportFooter', ({ environmentInfo }) => {
-    const { extensionVersion, browserSpec, axeCoreVersion } = environmentInfo;
+export const ReportFooter = NamedFC<ReportFooterProps>('ReportFooter', ({ environmentInfo, footerText: Text }) => {
```

Comment:
footerText property is not used

Reply:
It's used, it's aliased to Text - It has to be uppercase to render.

Reply:
nice. i thought it was the type of footerText.